src/ir_Mitsubishi.h

```diff
@@ -84,8 +84,16 @@ union Mitsubishi144Protocol{
     uint8_t Timer       :3;
     uint8_t WeeklyTimer :1;
     uint8_t             :4;
-    // Byte 14~16
-    uint8_t pad1[3];
+    // Byte 14
+    uint8_t          :4;
+    uint8_t          :4;
+    // Byte 15
+    uint8_t          :4;
+    uint8_t          :4;
```

Comment:
For now these can be kept as `:8;` instead of double? `:4;` or is there some specific reason that it is split?